instrumentation/log4j/log4j-2.13.2/auto/src/main/java/io/opentelemetry/instrumentation/auto/log4j/v2_13_2/Log4j2MdcInstrumentation.java

```diff
@@ -40,6 +40,8 @@ public Log4j2MdcInstrumentation() {
 
   @Override
   public ElementMatcher<? super TypeDescription> typeMatcher() {
+    // need to instrument a class, not an interface (ContextDataProvider) - otherwise helper classes
```

Comment:
"match", not "instrument" :) As you say yourself in `transformers`: "Nothing to instrument"

Reply:
That's true 🤦 

Little things like that are the easiest to miss 😄